partner-portal/src/UI/src/app/shared/api/models/oidc-options.ts

```diff
@@ -0,0 +1,7 @@
+/* tslint:disable */
+/* eslint-disable */
+export interface OidcOptions {
```

Comment:
this will be deleted if you delete ConfigController

Reply:
Deleted the config controller